RHDEVDOCS-3358 - new content for dex and keycloak (#39282)
openshift/openshift-docs: modules/gitops-uninstall-keycloak.adoc

> [id="uninstalling-keycloak_{context}"]

Reviewer: ```suggestion
[id="gitops-uninstall-keycloak_{context}"]
```
The `id` value needs to be the same as the filename per the OCP docs guidelines: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#module-file-metadata

= Uninstalling Keycloak

> You can delete the Keycloak resources and its relevant configuration by removing the SSO field from Argo CD Custom Resource Spec. After removing, it looks like the following:

Reviewer 2: the `Argo CD` custom resource spec
After removing the SSO field, the spec looks like the following:

> You can delete the Keycloak resources and its relevant configuration by removing the SSO field from Argo CD Custom Resource Spec. After removing, the SSO field will look like as shown below.

Reviewer 3: Let's rephrase the last sentence to avoid future tense and the spacial "below." For accessibility, we try to avoid these visual works like "below, above" or colors, like, "the green arrow button." 

`After removing, the SSO field looks like the following:`

> You can delete the Keycloak resources and its relevant configuration by removing the SSO field from Argo CD Custom Resource Spec. After removing the SSO field, the spec looks like the following:

Reviewer: ```suggestion
You can delete the Keycloak resources and their relevant configurations by removing the `SSO` field from the Argo CD Custom Resource (CR) file. After you remove the `SSO` field, the values in the file look similar to the following:
```

[source,yaml]
----
  apiVersion: argoproj.io/v1alpha1
  kind: ArgoCD
  metadata:
    name: example-argocd
    labels:
      example: basic
  spec:
    server:
      route:
       enabled: true
----

[NOTE]
====

> A Keycloak application created by this feature is currently not persistent. In case of restarts, any additional configuration created in the Argo CD Keycloak realm is deleted.

Reviewer 2: by this feature/by this method?

> A Keycloak application created by this feature is currently not persistent. In case of restarts, any additional configuration created in ArgoCD Keycloak realm is deleted.

Reviewer 3: Missing article:

`In case of restarts, any additional configuration created in **the** Argo CD Keycloak realm is deleted.` 

Also, I think there should be a space between Argo CD. At least, you always have that space in all other instances when you mention it.

> Keycloak application created by this feature is currently not persistent. Incase of restarts, nny additional configuration created in ArgoCD Keycloak realm will be deleted.

Reviewer 3: Minor tweaks to the Note (small spelling and grammatical errors):

`A Keycloak application created by this feature is currently not persistent. In case of restarts, any additional configuration created in ArgoCD Keycloak realm is deleted.`

> A Keycloak application created by this method is currently not persistent. In case of restarts, any additional configuration created in the Argo CD Keycloak realm is deleted.

Reviewer: ```suggestion
A Keycloak application created by using this method is currently not persistent. Additional configurations created in the Argo CD Keycloak realm are deleted when the [server????] restarts.
```
`In case of restarts` is not clear. What is being restarted?

> A Keycloak application created by this method is currently not persistent. A Keycloak application created by using this method is currently not persistent. Additional configurations created in the Argo CD Keycloak realm are deleted when the server restarts.

Reviewer 2: Delete the first sentence, it is repeated, the second one is better as it uses-'by using'
